<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>thipage</class>
 <widget class="QDialog" name="thipage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>314</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 0, 127);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 900 12pt &quot;Segoe UI Black&quot;;</string>
   </property>
   <property name="text">
    <string>Choose Package</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>221</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 10pt &quot;Segoe UI&quot;;
background-color: rgb(226, 226, 226);</string>
   </property>
   <property name="title">
    <string>Packages</string>
   </property>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>400 TK/month</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>181</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>1000 TK/3 Months</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>191</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>1600 TK/ 6 Months</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>191</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>3500 TK/12 Months</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_next2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
